conversationgenome/utils/config.py

```diff
@@ -31,17 +33,20 @@
         print("bittensor not installed")
     bt = MockBt()
 
-logger = None
-try:
-    from loguru import logger
-except:
-    print("No loguru")
+#logger = None
+#try:
+    #from loguru import logger
+#except:
+    #print("No loguru")
```

Comment:
If there is a reason for keeping this, please add a comment to indicate why. Otherwise, delete.